drag_and_drop_v2/templates/html/js_templates.html

```diff
@@ -5,27 +5,33 @@
         width:{{ width_percent }}%;
         height:{{ height_percent }}%;">
```

Comment:
Since this is focusable, I'm wondering if we should give it aria-describedby="" that references the `p.description` element below.